README.md

```diff
@@ -4,9 +4,9 @@
 | [![][docs-stable-img]][docs-stable-url] [![][docs-latest-img]][docs-latest-url] |[![][license-img]][license-url]   | [![][travis-img]][travis-url] |
 
 [docs-stable-img]: https://img.shields.io/badge/docs-latest-blue.svg
```

Comment:
I think the badges should change, if am not wrong. Or am just tired, but I think we have the badges the other way round. Please correct me if am wrong.

Reply:
Looks like it's me who is too tired. Sorry for the confusion. I'll push a fix in a couple of minutes.

Reply:
OK, this should be fixed now. Apologies for the confusion.